kvmagent/kvmagent/plugins/ft_vm_fencer.py

```
@@ -60,7 +60,7 @@ def start_raid_heartbeat():
         def getstatusoutput(c):
             # type: (str) -> (int, str)
             start_time = time.time()
-            r, o = commands.getstatusoutput(c)
+            r, o = subprocess.getstatusoutput(c)
```

Ruff (0.15.2)

[error] Starting a process with a shell, possible injection detected

(S605)